web3-dev/smart-contracts/getting-started.rst

```diff
@@ -5,61 +5,64 @@
 Getting started
 ===============
 
-In this section you will learn how to compile your first Massa smart contract.
+In this section you will learn how to setup your smart contract development environment, how to deploy your
```

Comment:
```suggestion
In this section you will learn how to setup your smart-contract development environment, how to deploy your
```

Reply:
Ena said me no dash!  i changed all `smart-contract` to `smart contract`

Reply:
I think this one is not resolved afterall!